app/src/main/java/com/orange/ods/app/ui/components/snackbars/ComponentSnackbars.kt

```diff
@@ -65,13 +82,53 @@ fun ComponentSnackbars() {
                 trailing = OdsSwitchTrailing(checked = actionOnNewLineChecked, enabled = actionButtonChecked.value)
             )
         }) {
-        ComponentLaunchContentColumn(textRes = R.string.component_snackbar_customize, buttonLabelRes = R.string.component_snackbar_show) {
-            coroutineScope.launch {
-                bottomSheetScaffoldState.snackbarHostState.showSnackbar(
-                    message = snackbarMessage,
-                    actionLabel = if (actionButtonChecked.value) snackbarActionLabel else null,
+        Column(
+            modifier = Modifier.verticalScroll(rememberScrollState())
+        ) {
+            ComponentLaunchContentColumn(textRes = R.string.component_snackbar_customize, buttonLabelRes = R.string.component_snackbar_show) {
+                coroutineScope.launch {
+                    bottomSheetScaffoldState.snackbarHostState.showSnackbar(
+                        message = snackbarMessage,
+                        actionLabel = if (actionButtonChecked.value) snackbarActionLabel else null
+                    )
+                }
+            }
+
+            CodeImplementationColumn(modifier = Modifier.padding(horizontal = dimensionResource(id = R.dimen.screen_horizontal_margin))) {
+                OdsTextBody2(
+                    modifier = Modifier.padding(bottom = dimensionResource(id = R.dimen.spacing_xs)),
+                    text = stringResource(id = R.string.component_snackbar_code_first_step)
                 )
+                ComposableCode(
+                    name = "OdsSnackbarHost",
```

Comment:
```suggestion
                    name = OdsComponent.OdsSnackbarHost.name,
```

We will need to add the missing `@OdsComponentApi` annotation on `OdsSnackbarHost`.